### ISSUE TITLE
Ansible style guide: Names for tasks

### ISSUE THREAD

**Issue by ra-rau** [osism/documentation#371]
I think it is better for understanding and easy listening to name all tasks. Sometimes in the Osism repositories ansible tasks have names and sometimes not. For example here https://github.com/osism/container-image-ceph-ansible/blob/main/files/playbooks/master/ceph-purge-storage-node.yml:

Without name:
```yaml
  - ansible.builtin.import_role:
      name: ceph-defaults
```

With name:
```yaml
  - name: remove ceph data
    ansible.builtin.shell: rm -rf /var/lib/ceph/*
```
This is one example for many other.

**Comment by berendt** [comment]
Yes, please document that a name must always be set for our things.

However, we do not change this in the playbook from which the example is taken. This is copied from ceph-ansible as it is.
